libs/k8s-client/src/KubernetesApiClientFacade.php

```diff
@@ -71,8 +71,8 @@ public function get(string $resourceType, string $name, array $queries = [])
      *       new Pod(...),
      *     ])
      *
-     * @param array<ResourceType> $resources
-     * @return ResourceType[]
+     * @param array<Event|Pod|Secret> $resources
```

Comment:
Toto je asi to co to ti nefungovalo, že? To je škoda no

Reply:
No v PHPstan, nekde ty aliasy fungovaly, nekde ne. PHPStorm je nepochopila snad nikde :smile: